ts/demo-functions/src/kubeval_test.ts

```diff
@@ -0,0 +1,44 @@
+import { Configs, kubernetesObjectResult, TestRunner } from 'kpt-functions';
+import { kubeval } from './kubeval';
+import { Namespace, PodTemplateSpec } from './gen/io.k8s.api.core.v1';
+import { Deployment, DeploymentSpec } from './gen/io.k8s.api.apps.v1';
+import {
+  LabelSelector,
+  ObjectMeta,
+} from './gen/io.k8s.apimachinery.pkg.apis.meta.v1';
+
+const RUNNER = new TestRunner(kubeval);
+
+describe('kubeval', () => {
+  it('handles objects without errors', async () => {
+    await RUNNER.assert(
+      new Configs([Namespace.named('something')]),
+      new Configs([Namespace.named('something')])
+    );
+  });
+  it('reacts on errors', async () => {
```

Comment:
This test is failing because it invokes kubeval process, requiring the kubeval and its dependencies to be installed. It's more of an integration or e2e testing than unit testing. I see two options here:

1) Mock out the invocation to kubeval and return and hardcode some expected output from kubeval: https://volaresystems.com/technical-posts/mocking-calls-with-jasmine

Value of this is questionable since it's not really testing how kubeval is parsing input and behaving.

2) If kubeval can be hermetic (doesn't require talking to api server), we can look into running this type tests inside the function docker container instead of directly on the host. But this option is more involved and requires changes to the CI/CD flows.

To not get blocked, I suggest either doing #1 or comment out the unit test for now. I'll then file a issue to revisit #2 as a general mechanism for testing.